settings.gradle

```diff
@@ -43,13 +43,6 @@ gradle.ext.scalafmt = [
     config: new File(rootProject.projectDir, '.scalafmt.conf')
 ]
 
-gradle.ext.scoverage = [
```

Comment:
Now scoverage plugin internally adapts the classpath with required dependencies. So no need to explicitly add them